index.html

```diff
@@ -53,6 +53,168 @@ <h1>
+                        <li>Ruby on Rails</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                        <li>HTML</li>
+                    </ul>
+                    <button>See Project</button>
+                </div>
+            </div>
+            <div class="card">
+                <div class="card_image">
+                    <img src="logo/Img Placeholder.png" alt="Image placeholder">
+                </div>
+                <div class="card_content">
+                    <h3>Multi-Post Stories <br> Gain+Glory</h3>
+                    <ul>
+                        <li>Ruby on Rails</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                        <li>HTML</li>
+                    </ul>
+                    <button>See Project</button>
+                </div>
+            </div>
+        </div>
+    </section>
+    <!-- About -->
+    <section class="about">
+        <div class="about_content">
+            <div>
+                <div class="illustration1">
+                    <h3>About Me</h3>
+                    <img src="logo/illustration 1- about me-mobile.png" alt="">
+                </div>
+                <p> I can help you build a product , feature or website Look through some of my work and experience! If
+                    you like what you see and have a project you need coded,
+                    don't hesitate to contact me.</p>
+                <button>Get my Resume</button>
+            </div>
+            <div class="about_boxes">
+                <div class="box">
+                    <div> <img src="logo/icon-languages.png" alt="Languages"></div>
+                    <h2>Languages</h2>
+                    <ul>
+                        <li>JavaScript</li>
+                        <li>Ruby</li>
+                        <li>HTML</li>
+                        <li>CSS</li>
+                    </ul>
+                </div>
+                <div class="box">
+                    <div> <img src="logo/icon-frameworks.png" alt="Frameworks"></div>
+                    <h2>Frameworks</h2>
+                    <ul>
+                        <li>React.js</li>
+                        <li>Ruby on Rails</li>
+                        <li>RSpec</li>
+                    </ul>
+                    <ul>
+                        <li>Capybara</li>
+                        <li>Selenium</li>
+                    </ul>
+                </div>
+                <div class="box">
+                    <div> <img src="logo/icon-skills.png" alt="Skils"></div>
+                    <h2>Skills</h2>
+                    <ul>
+                        <li>Database Management</li>
+                        <li>Version Control</li>
+
+                    </ul>
+                    <ul>
+                        <li>CLI</li><li>Web Development</li> <li>API Design</li>
+                  
+                    </ul>
+                </div>
+            </div>
```

Comment:
- [ ] As stated in the project requirement You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). So Kindly adjust the alignment of the the technology categories on the right side just like the figma design.
- Your technology categories design Look likes this :- 

![figma design](https://github.com/Cheptoo21/Portfolio-setup-and-mobile-first/assets/93598189/a30b1a2b-25bb-4844-8892-d47a835dd55b)

- The Figma design Look likes this :- 

![language](https://github.com/Cheptoo21/Portfolio-setup-and-mobile-first/assets/93598189/c108767b-0b91-469b-90dd-b085e63e65ff)